library/gui/key_bindings/src/main/java/org/quiltmc/qsl/key/bindings/mixin/client/KeyBindingEntryMixin.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2021 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.key.bindings.mixin.client;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyArg;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.widget.ButtonWidget;
+import net.minecraft.client.gui.widget.option.KeyBindingListWidget;
+import net.minecraft.client.gui.widget.option.KeyBindingListWidget.KeyBindingEntry;
+import net.minecraft.client.option.KeyBinding;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Formatting;
+
+import org.quiltmc.qsl.key.bindings.impl.ConflictTooltipOwner;
+
+@Environment(EnvType.CLIENT)
+@Mixin(KeyBindingEntry.class)
+public abstract class KeyBindingEntryMixin extends KeyBindingListWidget.Entry implements ConflictTooltipOwner {
```

Comment:
I would add a way to show that a keybinding is disabled. I'm sure there is a texture that would work for this

Reply:
When a keybind's disabled, not only it won't work, but it'll be hidden in the game; It's that way because it would be perfect for the usecase of disabling a keybind in-game without requiring a restart (ex: Ok Zoomer's extra keybinds).

However, thinking about it, perhaps I should split the `enabled` state into `enabled` and `visible`; I'll see what I can do, since there's still room for major changes like that

Reply:
Ah ok. I thought it would still show up, but have no effect in game. makes sense now.

Reply:
After some thought, having a separate state for only disabling it functionally doesn't really make sense; Mods can already do that themselves